mathesar_ui/src/systems/table-view/table-inspector/column/ColumnOptions.svelte

```diff
@@ -79,37 +79,55 @@
   }
 </script>
 
-<div class="properties-container">
-  <!--
-    TODO Once we have a DropdownMenu component, make this option
-    disabled if the column is a primary key.
-  -->
-  <Button appearance="plain" on:click={toggleAllowNull}>
-    {#if isRequestingToggleAllowNull}
+<div class="column-options">
+  <button class="passthrough" on:click={toggleAllowDuplicates}>
+    {#if isRequestingToggleAllowDuplicates}
       <Icon class="opt" {...iconLoading} />
     {:else}
-      <span class="opt"><Checkbox checked={allowsNull} /></span>
+      <span class="opt"><Checkbox checked={!allowsDuplicates} /></span>
     {/if}
-    <span>Allow NULL</span>
```

Comment:
Why is a Checkbox enclosed within button here (and in a few other places)?

Use the `LabelledInput` component if the purpose here is to show the label inline.